vm/src/obj/objbytes.rs

```diff
@@ -82,6 +86,66 @@ fn bytes_eq(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(vm.ctx.new_bool(result))
 }
 
+fn bytes_ge(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(a, Some(vm.ctx.bytes_type())), (b, None)]
+    );
+
+    let result = if objtype::isinstance(b, &vm.ctx.bytes_type()) {
+        get_value(a).to_vec() >= get_value(b).to_vec()
+    } else {
+        false
```

Comment:
Should this always be false, or should it raise a typeerror? In other words, could you add a test for `bytes() > 2`?

Reply:
You are right! Fix this.